### ISSUE TITLE
Can't add decks in Safari on a Mac

### ISSUE THREAD

**Issue by vatrachos** [seastan/ringsdb#33]
...but it works fine in Chrome.

**Comment by seastan** [comment]
What do you mean by "add" decks?

**Comment by vatrachos** [comment]
I am sorry for being vague! Adding decks in the quest log. Nothing happens when I click “Select deck.”

**Comment by seastan** [comment]
And what if you go to a deck page and select "Log a Quest" with that deck? Does it auto-fill the deck for you once you arrive at the log page?

**Comment by vatrachos** [comment]
If I do that, it the wheel spins forever like this… It does not populate.

**Comment by CloudNiner** [comment]
In both of the following cases (which are described above):
1. My Collection -> Quest Logs -> Log a Quest
1. My Collection -> Decks -> Select a Deck -> Log a Quest

Safari throws `InvalidStateError: The object is in an invalid state` in this code:
```javascript
    ui.init_date_picker = function() {
        var d = $('#date');
        if (!d.val()) {
            d[0].valueAsDate = new Date();  // <--- error here
        }
        d.on('input', ui.set_questlog_name).trigger('input');
    };
```

Stack trace in Safari console:
<img width="362" alt="Screen Shot 2020-10-20 at 11 40 55 AM" src="https://user-images.githubusercontent.com/1818302/96610274-71afb880-12c9-11eb-9ccc-65c5bf247cbd.png">

**Comment by seastan** [comment]
@CloudNiner Does method 2 work if you select multiple decks?

**Comment by CloudNiner** [comment]
No. Same error if I go My Collection -> Decks -> Select Multiple -> With Selection -> Log a Quest

**Comment by seastan** [comment]
Hmmm, ok thanks.
